src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts

```diff
@@ -0,0 +1,39 @@
+import { changedCookieHeaderValue } from './changedCookieHeaderValue'
+
+const cookieName = 'my-testCookie'
+const cookieValue = 'abc-123&;/abc-_'
+
+describe('changedCookieHeaderValue', () => {
+  it('should not contain the expiration attribute', () => {
+    expect(changedCookieHeaderValue(cookieName, cookieValue)).not.toContain(
+      '; expires',
+    )
+  })
+  it('has a value', () => {
+    expect(changedCookieHeaderValue(cookieName, cookieValue)).toMatch(
+      new RegExp(`^${cookieName}=.+;`),
+    )
+  })
+  it.todo('url encodes the value')
```

Comment:
is it a todo for this PR?

Reply:
I'd like to leave it for the future. The JWT tokens are already URL encoded, so this is not really needed for this library. But it's potentially an issue if the library evolves.